doc/operators/ledger.rst

```diff
@@ -15,15 +15,14 @@ Nodes can be configured to store their ledger under a particular directory with
 File layout
 -----------
 
-The ledger directory contains a series of files. File size is controlled by the ``--ledger-chunk-min-bytes`` command line option.
+The ledger directory contains a series of files. File size is controlled by the ``--ledger-chunk-bytes`` command line option.
```

Comment:
```suggestion
The ledger directory contains a series of files. File size is controlled by the ``--ledger-chunk-bytes`` command line option. Note that this size is a target. Because transactions may be arbitrarily large and cannot be split across chunks, it may be exceeded if a chunk contains a transaction approaching or exceeding ``--ledger-chunk-bytes``.
```